admin/cluster_containers_setup.py

```diff
@@ -0,0 +1,324 @@
+# Copyright ClusterHQ Inc.  See LICENSE file for details.
+import yaml
+import os
+import sys
+
+from copy import deepcopy
+from json import dumps
+from itertools import repeat
+from treq import json_content
```

Comment:
Mix these standard lib imports into the first group, except `treq` which is a third-party library.